package-lock.json

```diff
@@ -29,17 +37,26 @@
     },
     "@sinonjs/formatio": {
       "version": "2.0.0",
-      "resolved": "https://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
```

Comment:
@AndreyMitsyk - any idea why the paths in your `package-lock.json` are resolving as `http` instead of `https`?

Reply:
Don know, it was resolved automatically. @alex-margaritov do you have an idea why it happened?

Reply:
never mind. Regenerated file and now it looks fine.

Reply:
Great. Thanks @AndreyMitsyk - approving!